.github/assets/register-downstream-clusters.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+url="${1-172.18.0.1.omg.howdoi.website}"
+cluster="${2-k3d-k3s-second}"
+
+# hardcoded from .github/assets/token-ci.yml
+token="token-ci:zfllcbdr4677rkj4hmlr8rsmljg87l7874882928khlfs2pmmcq7l5"
```

Comment:
just curious if is it safe to have this token like this. Im guessing the environment is ephemeral, am I correct?

Reply:
Yes, it's all within the Github actions runner. If we self-host in the future, we will need to deal with isolation and clean up.